roles/docker/templates/daemon.json.j2

```diff
@@ -0,0 +1,15 @@
+{
+    "ip": "127.0.0.1",
+    "iptables": false,
+    "ipv6": false,
+    "storage-driver": "overlay2",
```

Comment:
@akebrissman This is the new way, as of CentOS 7.2 from what I've read. Used to always choose `device-mapper` driver before. Should probably be changed in all projects, with the addition of using XFS instead of EXT4 for the `/var/lib/docker` logical volume.

All and all.. I think this role should be pretty good for your garage Docker host (and elsewhere). Also relying on 1 year newer docker-ce (and -compose) version instead of clinging on to good-old `17.06.01` release.